doc/source/install/tarball.rst

```diff
@@ -10,14 +10,35 @@ for manual installation.
 You can download the appropriate tarball package from
 https://www.percona.com/downloads/Percona-XtraDB-Cluster-80
 
-There are multiple tarballs in the **Linux - Generic** section
-depending on the *OpenSSL* library available in your distribution:
+The **Linux - Generic** section has tarballs for each supported platform. The
+code name of the platform is part of the file name. For example, files ending in
+*buster.tar.gz* are built for |debian| buster, files ending in *el8.tar.gz* are
+built for CentOS 8:
```

Comment:
I am not sure that this is correct. Illia already sent email to experts regarding tarball names.
but if we talk about how it is implemented currently then it is correct